.github/workflows/ci.yml

```diff
@@ -85,6 +85,41 @@ jobs:
         chmod +x codecov
         ./codecov
 
+  #
+  # Android build job
+  #
+  android:
+    runs-on: macos-latest
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
+
+    # Definition of the build matrix
+    strategy:
+      matrix:
+        java: [ 11 ]
+        entry:
+          - { mock-maker: 'mock-maker-default', member-accessor: 'member-accessor-default' }
```

Comment:
Do you want to keep a 1-element matrix?

Reply:
Actually might want to introduce more API levels in the future, because they have different JVMs. See my comment on the issue, API 21 fails, while API 29 passes.

Reply:
Yeah, I think we could remove the mock-maker part

Reply:
I think we settled on level 26 as the minimum: https://github.com/mockito/mockito/blob/main/build.gradle#L102

Reply:
For reference the [official docs](https://developer.android.com/about/versions/oreo/android-8.0#art) only list some new APIs, so even Android 8 doesn't support Java 11. However, as I understand with [desugaring](https://developer.android.com/studio/write/java11-default-support-table) Java 11+ APIs can be used as low as API 1. This is of course not practical, I think an API 21 minimum would be [enough](https://apilevels.com/). That said [official full Java 11 support is only in 33](https://developer.android.com/about/versions/13/features#core-libraries).

Anyway, based on the info, starting with `android-api: [26,27,28,29,30,31,33]` would work.
([32 has no Java changes](https://developer.android.com/sdk/api_diff/32/changes), it was a "feature drop", not given a major version, if it works, it could be included though)

Reply:
Just to clarify, the matrix should only change the emulator version, the compile/target/minSdk should be hardcoded to highest/lowest values as [Google recommends it (TL;DR: we have ~1 year to use the latest target, otherwise cannot publish)](https://support.google.com/googleplay/android-developer/answer/11926878).